docs/api-reference/carto/carto-layer.md

```diff
@@ -69,6 +69,26 @@ function App({viewState}) {
 }
 ```
 
+### Raster data (experimental)
+
+The CARTO platform supports storing data in raster format. When the `CartoLayer` is created with type `MAP_TYPES.RASTER` data will be loaded as tiled raster data, and a layer derived from [`ColumnLayer`](../layers/column-layer.md) will be created and a subset of properties will be inherited.
+
+```js
+import DeckGL from '@deck.gl/react';
+import {CartoLayer, MAP_TYPES} from '@deck.gl/carto';
+
+function App({viewState}) {
+  const layer = new CartoLayer({
+    type: MAP_TYPES.RASTER,
+    connection: 'bigquery',
+    data: 'cartobq.testtables.raster',
+    getFillColor: d => [d.properties.temperature, 0, 0]
```

Comment:
I prefer to have here a real example raster using bands. `cartobq.testtables.raster` doesn't exist, I suggest to upload a temperature raster at `cartobq.deckgl.temperature_raster`

Reply:
Better at `cartobq.public_account.temperature_raster`

Reply:
Good plan - I will do once the API is deployed so I can test